### PR TITLE
Fix -414 Adding Settings Route Button inside the Login html

### LINKED ISSUE #414: Adding Settings Link inside the Login html
**I'm submitting a ...**  (check one with "x")
- [x] feature request

**Current behavior:**

There is no link or button which redirects to Settings.html

screenshot from 2018-10-05 12-05-52

**Expected behavior:**

**Steps to reproduce:**
GO to login.html

**Would you like to work on the issue?**

Please let us know if you can work on it or the issue should be assigned to someone else.
```
Yes

### PR DESCRIPTION
**Pull request by sk9331657**
Fixes #414 

#### Checklist

- [x] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia) and my PR follows them.
- [x] My branch is up-to-date with the Upstream `master` branch.

#### Short description of what this resolves:

ezgif com-video-to-gif 9

### CODE REVIEW

**Comment by sk9331657** [comment]
@Akshat-Jain Please merge it

File: src/css/login.css [hunk 1]
```diff
@@ -90,4 +90,12 @@ label {
...
     margin: 4% auto auto;
+}
+#settings
+{
+    margin:auto;
+    width:50%;
+}
+button{
+    cursor: pointer;
 }
```
Comment (Akshat-Jain) [inline src/css/login.css]: Add EOF

File: src/js/login.js [hunk 2]
```diff
@@ -237,3 +238,8 @@ logoutButton.addEventListener("click", function logout(e) {
 });
 
 document.addEventListener("DOMContentLoaded", showLoggedInBlock(false));
+
+settings.addEventListener("click",()=>
+{
+    this.location.href="/options.html";
+});
```
Comment (Akshat-Jain) [inline src/js/login.js]: Add EOF

**Comment by sk9331657** [comment]
@stealthanthrax  @Akshat-Jain  Done Required Changes. PR ready to be merged. :smile:

**Pull request merged**